### PR TITLE
#855 mesheryctl cleanup to mesheryctl reset

### LINKED ISSUE #855: Move mesheryctl cleanup -> mesheryctl reset
**Current Behavior**
mesheryctl cleanup is the command to remove old meshery.yml file with default one.

**Desired Behavior**
A renaming of  `mesheryctl cleanup` to `mesheryctl reset` as the name of the command.
so that `mesheryctl reset` should be  the command to remove current meshery.yml file with default meshery.yml.

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

fixes #855

### CODE REVIEW

File: mesheryctl/cmd/root.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ Usage:
   mesheryctl [command]
 
 Available Commands:
-  cleanup     Clean up Meshery
+  reset       Clean up Meshery
```
Comment (kushthedude) [inline mesheryctl/cmd/root.go:39]: Change the description over here too!

File: meshes/meshops.pb.go [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // source: meshops.proto
 
+//lint:file-ignore SA1019 Ignore the file until Google's protocol buffer supports gRPC Service Definitions. For more info see : https://github.com/layer5io/meshery/pull/848#issuecomment-613965380
```
Comment (kushthedude) [inline meshes/meshops.pb.go:4]: Also, this change is already done in a PR #848, please revert it.

File: mesheryctl/cmd/root.go (hunk 1 above)
Comment (gunjanmimo) [reply mesheryctl/cmd/root.go:39]: yah sure.. thank you

File: meshes/meshops.pb.go (hunk 2 above)
Comment (gunjanmimo) [reply meshes/meshops.pb.go:4]: I think this is my mistake... I will sync my branch

**Comment by gunjanmimo** [comment]
@kushthedude may you take a quick look?

File: meshes/meshops.pb.go (hunk 2 above)
Comment (kushthedude) [reply meshes/meshops.pb.go:3]: Same issue!

File: go.mod [hunk 3]
```diff
@@ -13,7 +13,7 @@ require (
 	github.com/go-redis/redis v6.15.5+incompatible // indirect
 	github.com/gofrs/uuid v3.2.0+incompatible
 	github.com/gogo/protobuf v1.3.1 // indirect
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.0
```
Comment (kushthedude) [inline go.mod:16]: Change already done in #848 !

File: go.sum [hunk 4]
```diff
@@ -94,8 +94,12 @@ github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5y
 github.com/golang/protobuf v1.3.1/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
 github.com/golang/protobuf v1.3.2/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
 github.com/golang/protobuf v1.3.3/go.mod h1:vzj43D7+SQXF/4pzW/hwtAqwc6iTitCiVSaWz5lYuqw=
-github.com/golang/protobuf v1.3.5 h1:F768QJ1E9tib+q5Sc8MkdJi1RxLTbRcTf8LJV56aRls=
-github.com/golang/protobuf v1.3.5/go.mod h1:6O5/vntMXwX2lRkT1hjjk0nAC1IDOTvTlVgjlRvqsdk=
+github.com/golang/protobuf v1.4.0-rc.1/go.mod h1:ceaxUfeHdC40wWswd/P6IGgMaK3YpKi5j83Wpe3EHw8=
```
Comment (kushthedude) [inline go.sum:97]: Change already done in #848 !

File: meshes/meshops.pb.go (hunk 2 above)
Comment (gunjanmimo) [reply meshes/meshops.pb.go:3]: should I remove this line no4  from my pr ??

**Comment by leecalcote** [comment]
@gunjanmimo will you have a look at the conflict in cleanup.go? Let’s see if we can get this one across the finish line. Thanks for taking it on. 👍

**Comment by gunjanmimo** [comment]
@leecalcote sir and @ kush may you take a look now?

File: mesheryctl/cmd/cleanup.go [hunk 5]
```diff
@@ -22,8 +22,8 @@ import (
 // cleanupCmd represents the cleanup command
 var cleanupCmd = &cobra.Command{
 	Use:   "cleanup",
-	Short: "Cleanup Meshery's configuration",
-	Long:  `Reset Meshery to it's default configuration.`,
+	Short: "Cleaning up Meshery's configuration",
```
Comment (thakursaurabh1998) [inline mesheryctl/cmd/cleanup.go:25]: Aren't we moving to change this command to reset ? @leecalcote

Comment (leecalcote) [reply mesheryctl/cmd/cleanup.go:25]: @thakursaurabh1998 yes, that's right. 👍

**Pull request closed without merging**